apps/web/public/static/locales/en/common.json

```diff
@@ -217,6 +217,9 @@
   "already_have_an_account": "Already have an account?",
   "create_account": "Create Account",
   "confirm_password": "Confirm password",
+  "confirm_google_auth_change": "This will change the way you log in",
+  "confirm_google_auth_email_change": "Changing the email address will disconnect your Google account to log in to Cal.com. We will ask you to verify your new email address. Moving forward, you will be logged out and use your new email address to log in instead of your Google Account after setting your password by following the instructions that will be sent to your mail.",
```

Comment:
I know this is intended for Google primarily but I recommend making this message more generic so other auth providers (on revert) can also show this message.